### PR TITLE
More examples with volumes

### PR DESCRIPTION
**Pull request by mathieu-benoit**
Continuity of https://github.com/humanitec-architecture/example-library/pull/105 + add more use cases based on https://github.com/Humanitec-DemoOrg/volumes.

`volumes-static-provisioning` is now `volumes` to be consistent and grouped in the library categories.

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
Ready for your review, thanks!

File: resource-definitions/template-driver/volumes/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This section contains Resource Definitions example for handling Kubernetes [Volumes](https://kubernetes.io/docs/concepts/storage/volumes) by using the [`template`](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) Driver to configure your own `PersistentVolume` implementation. You can [see this other example](https://developer.humanitec.com/examples/resource-definitions/volume-pvc/volumes/) if you want to use the `volume-pvc` Driver.
```
Comment (chrishumanitec) [inline resource-definitions/template-driver/volumes/README.md:1]:
```suggestion
This section contains Resource Definition examples for handling Kubernetes [Volumes](https://kubernetes.io/docs/concepts/storage/volumes) by using the [`template`](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) Driver to configure your own `PersistentVolume` implementation. You can [see this other example](https://developer.humanitec.com/examples/resource-definitions/volume-pvc/volumes/) if you want to use the `volume-pvc` Driver.
```

File: resource-definitions/template-driver/volumes/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (chrishumanitec) [inline resource-definitions/template-driver/volumes/README.md:2]: Shouldn't this also mention the `emotyDir` example too?

**Review by chrishumanitec — COMMENTED** [review]
One typeo, and an apparently missing section in the README in volumes for emptyDir

File: resource-definitions/template-driver/volumes/README.md (hunk 2 above)
Comment (mathieu-benoit) [reply resource-definitions/template-driver/volumes/README.md:6]: Good point, I tried something, but I'm sure @TobiasBabin will make this even better.

File: resource-definitions/template-driver/volumes/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This section contains Resource Definition examples for handling Kubernetes [Volumes](https://kubernetes.io/docs/concepts/storage/volumes) by using the [`template`](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) Driver to configure your own `PersistentVolume` implementation. You can [see this other example](https://developer.humanitec.com/examples/resource-definitions/volume-pvc/volumes/) if you want to use the `volume-pvc` Driver.
+
+You will find two examples:
+- `volume-emptydir` - in order to inject an `emptyDir` `volume` in a Workload for any request of a `volume` resource with the `class` `ephemeral`.
+- `volume-nfs` - in order to create the associated `PersistentVolumeClaim`, `PersistentVolume` and `volume` in a Workload for any request of a `volume` resource with the `class` `nfs`.
+
+You can find a Score file example using the `volume` resource type [here](https://developer.humanitec.com/examples/score/horizontalpodautoscaler/).
```
Comment (TobiasBabin) [inline resource-definitions/template-driver/volumes/README.md:7]:
```suggestion
You can find a Score file example using the `volume` resource type [here](https://developer.humanitec.com/examples/score/volumes/).
```

File: score/volumes/README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 Define volumes and volume mounts for a Workload.
 
-Volume mounts are declared in the `volumes` section of the Score file. There should be a resource of type `volume` backing the volume.
+Volume mounts are declared in the `volumes` section of the Score file. There should be a resource of type `volume` defined in your resource definitions, [see examples here](https://developer.humanitec.com/examples/resource-definitions/volume-pvc/volumes/).
```
Comment (TobiasBabin) [inline score/volumes/README.md:3]: This link will capture all Resource Definition examples containing a `type: volume`, regardless of its Driver.

```suggestion
Volume mounts are declared in the `volumes` section of the Score file. There should be a resource of type `volume` defined in your resource definitions, [see examples here](https://developer.humanitec.com/examples/resource-definitions?types=volume).
```

**Review by TobiasBabin — APPROVED** [review]
LGTM!

**Pull request merged**